packages/now-cli/src/util/alias/assign-alias.ts

```diff
@@ -21,71 +12,11 @@ export default async function assignAlias(
   client: Client,
   deployment: Deployment,
   alias: string,
-  contextName: string,
-  noVerify: boolean
+  contextName: string
 ) {
-  const prevAlias = await findAliasByAliasOrId(output, client, alias);
   let externalDomain = false;
 
-  // If there was a previous deployment, we should fetch it to scale and downscale later
-  let prevDeployment = await getDeploymentFromAlias(
-    client,
-    contextName,
-    prevAlias,
-    deployment
-  );
-
-  // If there is an alias laying around that points to a deleted
-  // deployment, we need to account for it here.
-  if (prevDeployment instanceof ERRORS.DeploymentNotFound) {
-    prevDeployment = null;
-  }
```

Comment:
Are you sure this won't return DeploymentNotFound error for v2?

What about it `prevDeployment` is an error?

Reply:
`prevDeployment` was only used for `now alias` without any arguments, which has been deprecated a while ago. So there is no such thing as a previous deployment anymore.